src/herbpy/tsr/fuze.py

```diff
@@ -0,0 +1,55 @@
+import numpy
+from prpy.tsr.tsrlibrary import TSRFactory
+from prpy.tsr.tsr import *
+
+@TSRFactory('herb', 'fuze_bottle', 'point')
+def point_at_obj(robot, bottle, manip=None):
+    '''
+    @param robot The robot performing the point
+    @param bottle The bottle to point
+    @param manip The manipulator to perform the grasp. if None 
+        the active manipulator on the robot is used
+    '''
+
+    if manip is None:
+        manip_idx = robot.GetActiveManipulatorIndex()
+    else:
+        with manip.GetRobot():
+            manip.SetActive()
+            manip_idx = manip.GetRobot().GetActiveManipulatorIndex()
+		
+    # compute T_ow
+    T0_w_0 = bottle.GetTransform()
+    T0_w_1 = numpy.identity(4)
+
+    # compute T_we. Tuned once and used forever onward 
+    # Insures pointer points at object with right arm
+    arm_start =  numpy.array([[ 0.07277, -0.71135,  0.69905,  0.58575],
+       			      [ 0.03360,  0.70226,  0.71111, -0.44275],
+    		              [-0.99678, -0.02825,  0.07501,  0.95686],
+ 			      [ 0.     ,  0.     ,  0.     ,  1.     ]])
+		
+    base_obj_trans = numpy.array([[1, 0, 0, 1.0],
+                                  [0, 1, 0, 0.0],
+                                  [0, 0, 1, 1.0],
+                                  [0, 0, 0, 1.0]])
+
+    TW_e_0 = numpy.dot(numpy.linalg.inv(base_obj_trans), arm_start)
```

Comment:
Ok, I think a lot of confusion would be mitigated by just performing this multiplication:
```numpy.dot(numpy.linalg.inv(base_obj_trans), arm_start)```
once offline and then putting the result in here.  So instead of arm_start or base_obj_trans you just have:

```
Tw_e_0 = numpy.array([[ 0.07277, -0.71135,  0.69905, -0.41425],
       [ 0.0336 ,  0.70226,  0.71111, -0.44275],
       [-0.99678, -0.02825,  0.07501, -0.04314],
       [ 0.     ,  0.     ,  0.     ,  1.     ]])
```
Also, we should try this for the left arm too.  I can't quite remember how the two end-effector frames are setup relative to the hands, but there is a chance this is valid for both - if each hand is oriented the same relative to its associated end-effector transform.